tensorpipe/test/channel/channel_test.cc

```diff
@@ -229,8 +229,12 @@ TEST_P(ChannelTest, SendTensorsBothWays) {
         // Perform send.
         {
           std::future<std::tuple<Error, TDescriptor>> descriptorFuture;
-          std::tie(descriptorFuture, sendFuture) =
-              sendWithFuture(channel, sendData.data(), sendData.size());
+          std::tie(descriptorFuture, sendFuture) = sendWithFuture(
+              channel,
+              CpuTensor{
+                  .ptr = sendData.data(),
+                  .length = sendData.size(),
```

Comment:
Any reason to use `.ptr` and `.length` here where in other cases in this file you just passed them as "positional" args, not as "keyword" args (not sure what's the C++ terminology)? Again, I'd like to avoid inconsistencies.

Reply:
Agreed. I think it is called aggregate intialization and designated aggregate intialization respectively.
I think the designated version is more readable, but apparently it's C++20 so we might want to do without for now.